gradle.properties

```diff
@@ -4,9 +4,10 @@ micronautDocsVersion=1.0.23
 micrometerVersion=1.5.4
 validationVersion=2.0.1.Final
 jcacheVersion=1.1.1
-micronautVersion=1.3.5
+micronautVersion=1.3.7
 micronautTestVersion=1.2.2
 groovyVersion=2.5.11
+nettyVersion=4.1.51.Final
```

Comment:
This version contains some regressions so we can't use it. We should stick at this moment with `4.1.48.Final` as in micronaut-core